src/bentoml_cli/models.py

```diff
@@ -240,13 +241,14 @@ def export(model_tag: str, out_path: str) -> None:  # type: ignore (not accessed
         help="The name to give to the model in the BentoML store",
     )
     def import_from(model_path: str, name: str | None) -> None:  # type: ignore (not accessed)
-        """Import a previously exported Model archive file
+        """Import a previously exported Model archive file or a model from remote model store
 
         bentoml models import ./my_model.bentomodel
         bentoml models import s3://mybucket/models/my_model.bentomodel
-        bentoml models import hf:t5-base --name myModelName
+        bentoml models import https://huggingface.co/t5-base --name myModelName
         """
-        bentomodel = import_model(model_path, name=name)
+        BENTOML_IMPORT_FRAMEWORK = os.environ.get("BENTOML_IMPORT_FRAMEWORK", None)
+        bentomodel = import_model(model_path, framework=BENTOML_IMPORT_FRAMEWORK)  # type: ignore
```

Comment:
Why the `# type: ignore` here?

Reply:
Ah there was a mismatch for the framework param since BENTOML_IMPORT_FRAMEWORK could be any str and framework is only 'transformers|diffusers'. Think I just used the quick fix tool suggested by vscode. Shouldn't be a big deal. I will remove the `# type: ignore`

Reply:
you can make the framework literal type